docs/apache-airflow/installation/upgrading.rst

```diff
@@ -79,6 +79,20 @@ table or rename it or move it to another database using those tools. If you don'
 can use the ``airflow db shell`` command - this will drop you in the db shell tool for your database and you
 will be able to both inspect and delete the table.
 
+Alternatively, you can drop the table by following the steps below:
+
+
+1. Exec into any of the Airflow pods - webserver or scheduler: ``kubectl exec -it <your-webserver-pod> python``
```

Comment:
The command example here is out of place since it is specific to Kubernetes, while the rest of the document does not make the assumption, and already assumes you have hooked into the machine/container/pod/whatever. I would only add the second bullet point.

Reply:
I'd rather give some more examples - kubernetes, docker, docker-compose and tell them this is an example and that they should find out what they should do in their environment.

 I think our assumption is that this documentation is used by  users who had never done low-level DB operations with Airflow DB, and they likely have no idea how to get to their containers. Giving them at least a handy example they could follow and at least the starting point that will allow them to ask the right google question to dig deeper is useful.

Reply:
I think this is just need to be in a note block that mention this is K8s specific

Reply:
Absolutely

Reply:
The note block makes sense